### ISSUE TITLE
SWProxy 0.99.1 - Compiled OSX not working

### ISSUE THREAD

**Issue by stanleykylee** [kakaroto/SWProxy#55]
The compiled zip package is missing the plugins folder.

**Comment by stanleykylee** [comment]
Created a .command file which will:

1. Launch a terminal session
2. Change to the directory in which the script (.command and SWProxy) file is located
3. Launch SWProxy

'''
cd `dirname $0`
`pwd`/SWProxy
'''

Note:
OSX terminal by default will launch the terminal session to the user's home directory. Placing the plugins directory in the user's home directory will also work and not require the .command file.

**Comment by Fl3xX** [comment]
Hi, 

When i run the command i get this in my terminal, can you give me a hand..?

Error loading Python lib '/var/folders/g6/g6xWIqVUGuO6CmDUxKu2Rk+++TI/-Tmp-/_MEI7ueMOb/Python': dlopen(/var/folders/g6/g6xWIqVUGuO6CmDUxKu2Rk+++TI/-Tmp-/_MEI7ueMOb/Python, 10): Symbol not found: ___sincos_stret
  Referenced from: /var/folders/g6/g6xWIqVUGuO6CmDUxKu2Rk+++TI/-Tmp-/_MEI7ueMOb/Python
  Expected in: /usr/lib/libSystem.B.dylib
 in /var/folders/g6/g6xWIqVUGuO6CmDUxKu2Rk+++TI/-Tmp-/_MEI7ueMOb/Python
logout

Thank you

**Comment by stanleykylee** [comment]
Are the files extracted from the zip file before you run the .command file? It looks like you're trying to run it from a temporary folder?

**Comment by Fl3xX** [comment]
Yes i Unziped them and they are on my desktop right now. I tried to install the latest version of python but honestly i'm not comfortable on mac at all ... Doesn't have something to do with pythonlib ? 

Tell me if you need anything that can help.

**Comment by stanleykylee** [comment]
@Fl3xX I created a new issue for your problem.

**Comment by lstern** [comment]
can we close this ?

**Comment by stanleykylee** [comment]
@lstern, yes I guess? @kakaroto didn't like the solution I provided, so technically the issue is still there.

I could open a new issue that specifically refers to the double-clicking as the binary works fine.

**Comment by kakaroto** [comment]
Let's leave it open for now. I haven't actually tried it on mac or looked at how it runs at all (just what i understood from reading the issue comments), so until I can actually test it and assess what needs to be done (maybe @stanleykylee 's solution is right, I just misunderstood what he did), let's keep it open.

**Comment by stanleykylee** [comment]
@kakaroto no worries. From my understanding, it's more related to how OSX handles a terminal session when launched by double-clicking that is causing the issue (need for .command file). However, from seeing various issues coming up now, I may need to make my .command file more compatible with older versions of OSX.

In the middle of preparing for final paper and exams, so I'll be in and out for the next two weeks.

**Comment by aureus1** [comment]
Just pushed a fix for this to my fork (and pull requested).

OSX binary here: https://github.com/aureus1/SWProxy/releases/tag/OSX_Fix

**Comment by msteudel** [comment]
@aureus1 - Sorry I'm new to the whole SWProxy thing ... what is your fork do? I went to look on the github page but didn't see any readme ...

**Comment by aureus1** [comment]
It's basically just how github works - I can't directly make code changes
to SWParser as I'm not part of the project, but I can fork it (basically
make my own copy), make changes there and request that they get pulled into
the main version.

That's what I did here - I copied the project and fixed this issue.  The
"OSX Binary" link in my original post is a zip containing the version of
SWParser that should fix this issue, built for Mac OSX.

**Comment by Fl3xX** [comment]
Hi guys and thank you for the update and fix, but i have the exact same issue.. I get this when i try to run the swproxy or the .command. 
 
I'm running on OSX 10.6.8 and i installed Xcode .
I really need some help to translate what the console said me :) Thanks !

Error loading Python lib '/var/folders/g6/g6xWIqVUGuO6CmDUxKu2Rk+++TI/-Tmp-/_MEIW9psS2/Python': dlopen(/var/folders/g6/g6xWIqVUGuO6CmDUxKu2Rk+++TI/-Tmp-/_MEIW9psS2/Python, 10): Symbol not found: ___sincos_stret
  Referenced from: /var/folders/g6/g6xWIqVUGuO6CmDUxKu2Rk+++TI/-Tmp-/_MEIW9psS2/Python
  Expected in: /usr/lib/libSystem.B.dylib
 in /var/folders/g6/g6xWIqVUGuO6CmDUxKu2Rk+++TI/-Tmp-/_MEIW9psS2/Python
logout

**Comment by aureus1** [comment]
@Fl3xX You're going to need OSX 10.9 or later; the compiled version is not compatible with earlier versions.
